corehq/apps/reports/static/reports/js/bootstrap5/async.js

```diff
@@ -213,21 +216,21 @@ hqDefine("reports/js/bootstrap5/async", [
                         } else {
                             humanReadable = self.humanReadableErrors[data.status];
                         }
-                        self.loadingIssueModal.find('.report-error-status').html('<strong>' + data.status + '</strong> ' +
+                        self.loadingIssueModalElem.find('.report-error-status').html('<strong>' + data.status + '</strong> ' +
                             ((humanReadable) ? humanReadable : ""));
                         if (self.issueAttempts > 0) {
-                            self.loadingIssueModal.find('.btn-primary').button('fail');  /* todo B5: plugin:button */
+                            self.loadingIssueModalElem.find('.btn-primary').changeButtonState('fail');
                         }
                         self.issueAttempts += 1;
-                        self.loadingIssueModal.modal('show');  /* todo B5: plugin:modal */
+                        self.loadingIssueModal.show();
                     } else {
                         self.hqLoading = $(self.loaderClass);
                         self.hqLoading.find('h4').text(gettext("Loading Stopped"));
                         self.hqLoading.find('.js-loading-spinner').attr('style', 'visibility: hidden;');
                     }
                 },
                 beforeSend: function () {
-                    self.standardReport.filterSubmitButton.button('loading');  /* todo B5: plugin:button */
+                    self.standardReport.filterSubmitButton.changeButtonState('loading');
```

Comment:
Can you update the docs on `button` to let readers know about `changeButtonState`?

https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/hqwebapp/utils/bootstrap/changes_guide/js-button.md

Reply:
done https://github.com/dimagi/commcare-hq/pull/35685